src/shared/components/Partner/component.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { PartnerWrapper, ImageElement } from './styles';
+
+type PartnerProps = {
+  imageSrc: string;
+  imageAlt: string;
+};
+
+const Partner: React.FC<PartnerProps> = (props: PartnerProps) => {
```

Comment:
I propose here `({imageSrc, imageAlt}: PartnerProps)`
And after `src={imageSrc} alt={imageAlt}`

Reply:
done